### PR TITLE
add a script for bot stats

### PR DESCRIPTION
**Pull request by ullasbharadwaj**
Based on discussion on Discord, I am attaching the script for retrieving bot stats.
Please check the path to db, i have added a default path and also please feel free to improve it. Thank you for your efforts.

### CODE REVIEW

**Comment by titulebolide** [comment]
Thanks! I'll take care of it soon

**Comment by titulebolide** [comment]
What's left to do:
- [ ] Use the table.tabularize tables instead of prettytable
- [ ] Take care that long coin list will not trigger `Too long message` errors
- [ ] Use translation files for bot_stats

File: i18n/en.yml [hunk 1]
```diff
@@ -216,6 +216,7 @@ keyboard:
   trade_history: "⌛ Trade History"
   panic: "🚨 Panic button"
   check_status: "🔍 Check bot status"
+  bot_stats: "Bot stats"
```
Comment (ullasbharadwaj) [inline i18n/en.yml:219]: Wouldn't it be nice to have an icon beside Bot stats? then everything would be uniform.

Comment (titulebolide) [reply i18n/en.yml:219]: Yes absolutely, it is just for now a placeholder, I'll be working on formating everything nicely soon

**Comment by titulebolide** [comment]
Closing in favor of #213

**Pull request closed without merging**